pkgs/by-name/mo/moneydance/package.nix

```
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchzip, makeWrapper, openjdk21, openjfx21, }:
+let jdk = openjdk21.override { enableJavaFX = true; };
+in stdenv.mkDerivation (finalAttrs: {
+  pname = "moneydance";
+  version = "2023.3_5064";
+
+  src = fetchzip {
+    url = "https://infinitekind.com/stabledl/${finalAttrs.version}/moneydance-linux.tar.gz";
+    hash = "sha256-jHr1V/gV1seenw2Q0/G405lTiabEYEsOS8p/XyByrtM=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ jdk openjfx21 ];
+
+  dontConfigure = true;
+  dontUnpack = true;
+  dontBuild = true;
+  dontFixup = true;
+
+  # N.B. The quotes around the classpath value protect the `*` from shell
+  # expansion. The JVM interprets a classpath value like `dir/*` specially,
+  # adding every JAR file in `dir` to the classpath.
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/libexec $out/bin
+    cp -p $src/lib/* $out/libexec/
+    makeWrapper ${jdk}/bin/java $out/bin/moneydance \
+      --add-flags -client \
+      --add-flags --add-modules \
+      --add-flags javafx.swing,javafx.controls,javafx.graphics \
+      --add-flags -classpath \
+      --add-flags "'${placeholder "out"}/libexec/*'" \
+      --add-flags Moneydance
+
+    runHook postInstall
+  '';
+
+  passthru = { inherit jdk; };
+
+  meta = {
+    homepage = "https://infinitekind.com/moneydance";
+    description =
+      "Moneydance is an easy to use and full-featured personal finance app that doesn't compromise your privacy";
```

See [the documentation](https://nixos.org/manual/nixpkgs/unstable/#var-meta-description), you shouldn't repeat the name of package in the description.

```suggestion
      "An easy to use and full-featured personal finance app that doesn't compromise your privacy";
```

---

Very good; done. Thanks very much for the review.